assets/js/phoenix_live_view.js

```diff
@@ -511,11 +511,36 @@ export class LiveSocket {
     for(let type of ["change", "input"]){
       this.on(type, e => {
         let input = e.target
-        let key = input.type === "checkbox" ? "checked" : "value"
-        if(this.prevInput === input && this.prevValue === input[key]){ return }
+
+        let key
+        if (input.type === "checkbox") {
+          key = "checked"
+        }
+        else if (input.type === "select-multiple") {
+          key = "selectedOptions"
+        }
+        else {
+          key = "value"
+        }
+
+        let value
+        // Handle multiselect values
+        if (input[key] instanceof HTMLCollection) {
+          let values = []
+          for (let item of input[key]) {
```

Comment:
ie11 compat issue?  Looks like there is an existing one but I thought ie11 support was important for this pkg...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of